pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -1383,7 +1383,8 @@ public void closeFailed(ManagedLedgerException exception, Object ctx) {
             log.debug("[{}] Checking replication status", name);
         }
 
-        List<String> configuredClusters = topicPolicies.getReplicationClusters().get();
+        List<String> configuredClusters = topicPolicies.getReplicationClusters().get() == null
```

Comment:
This could not be null if the topic is properly initialized.

Reply:
There was null checker in previous code

Reply:
> There was null checker in previous code


It's not the same implementation.
After looking into the codes, it's possible that this policy could be null if the namespace policy is invalid or deleted somehow.

A better solution would be add the following code in `org.apache.pulsar.broker.service.AbstractTopic#updateTopicPolicyByBrokerConfig`:
`topicPolicies.getReplicationClusters().updateBrokerValue(Collections.emptyList());`

Reply:
+1